src/libs/ajax/leonardo/Cookies.ts

```diff
@@ -0,0 +1,35 @@
+import { withAuthSession } from 'src/auth/auth-options';
+import { fetchLeoOnce } from 'src/libs/ajax/fetch/fetchLeoOnce';
+
+import { LeoCookiesDataClient, makeLeoCookiesDataClient } from './LeoCookiesDataClient';
+
+const makeCookiesHelper = (deps: CookiesHelperDeps) => (signal?: AbortSignal) => {
+  const { api } = deps;
+
+  const cookiesHelper = {
+    // setAzureCookie: async (proxyUrl: string): Promise<void> => {
```

Comment:
lets have a //TODO to explain why these two functions are here but commented out